ext/hal/nordic/Kconfig

```diff
@@ -4,12 +4,8 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-config HAS_NORDIC_MDK
-	bool
-
-config HAS_NORDIC_HAL
+config HAS_NORDIC_DRIVERS
 	bool
```

Comment:
I'm not sure what NORDIC_DRIVERS is, but shouldn't it be selecting NRFX?

Reply:
Right now it is only "nrf_drv_radio802154" in ext/hal/nordic/drivers.
NRFX is selected in nrf5x/Kconfig.series. Isn't it enough, or could that be wrong?

Reply:
It's fine.